crates/subspace-node/src/bin/subspace-node.rs

```diff
@@ -325,6 +325,22 @@ fn main() -> Result<(), Error> {
                         &config,
                         frame_benchmarking_cli::SUBSTRATE_REFERENCE_HARDWARE.clone(),
                     ),
+                    BenchmarkCmd::Extrinsic(_cmd) => {
+                        todo!("Not implemented")
+                        // let PartialComponents { client, .. } =
```

Comment:
Shouldn't we remove the commented code here?

Reply:
This code comes from Substrate's node template, I left it so whoever will look at it later will be able to fix it quicker. It almost compiles, but there are a few things like `RemarkBuilder` that do not exist in our codebase.